cargo/cargo_build_script_runner/lib.rs

```diff
@@ -187,7 +191,13 @@ impl BuildScriptOutput {
                 BuildScriptOutput::Flags(e) => compile_flags.push(e.to_owned()),
                 BuildScriptOutput::LinkArg(e) => compile_flags.push(format!("-Clink-arg={e}")),
                 BuildScriptOutput::LinkLib(e) => link_flags.push(format!("-l{e}")),
-                BuildScriptOutput::LinkSearch(e) => link_search_paths.push(format!("-L{e}")),
+                BuildScriptOutput::LinkSearch(e) => {
```

Comment:
Can you explain this change as well?

Reply:
@UebelAndre I had to make this change because one of the required C libraries was not getting found at the expected location. A prefix had to be added to the path but only for the power build and not the Intel build. Please allow me some time to recollect the name of the library and other the specifics.